test/custom-parser-async.test.js

```diff
@@ -24,12 +23,11 @@ test('contentTypeParser should add a custom async parser', t => {
     return res
   })
 
-  fastify.listen({ port: 0 }, err => {
-    t.error(err)
+  fastify.listen({ port: 0 }, async (err) => {
```

Comment:
we can't use `async` func as callback

async/await or callback styel

Reply:
Roger